miq_tools_services.gemspec

```diff
@@ -4,10 +4,14 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'miq_tools_services/version'
 
 Gem::Specification.new do |spec|
+
+  # Dynamically create the authors information {name => e-mail}
+  authors_hash = Hash[`git log --no-merges --reverse --format='%an,%ae'`.split("\n").uniq.collect {|i| i.split(",")}]
```

Comment:
I never liked this, personally.  I'd rather just list core committers...not everyone who contributes is an "author".  I think of authors as the people who own the project and who you could contact in case of problems